tests/post/steps/test_volume.py

```diff
@@ -97,10 +104,88 @@ def test_volume_delete_while_pending(host, teardown):
     pass
 
 
+@scenario(
+    "../features/volume.feature", "Test volume creation (sparseLoopDevice Block mode)"
+)
+def test_volume_creation_loop_block(host, teardown):
+    pass
+
+
+@scenario(
+    "../features/volume.feature", "Test volume creation (rawBlockDevice Block mode)"
+)
+def test_volume_creation_rawblock_block(host, teardown):
+    pass
+
+
+@scenario(
+    "../features/volume.feature", "Test volume creation (lvmLogicalVolume Block mode)"
+)
+def test_volume_creation_lvmlv_block(host, teardown):
+    pass
+
+
 # }}}
 # Given {{{
 
 
+@given("a device exist")
+def device_exist(context, host):
+    context["device_size"] = "20Gi"
+
+    with host.sudo():
+        cmd_ret = host.check_output("salt-call --out json --local temp.file")
+
+    sparse_file = json.loads(cmd_ret)["local"]
+
+    with host.sudo():
+        host.check_output(
+            "salt-call --local file.truncate {} {}".format(
+                sparse_file, _quantity_to_bytes(context["device_size"])
+            )
+        )
+        cmd_ret = host.check_output(
+            "salt-call --out=json --local cmd.run 'losetup -fP --show {}'".format(
+                sparse_file
+            )
+        )
```

Comment:
Why do we bother with Salt for this command?

Reply:
Because ..... I love salt ... Ok no I don't know why, I will change this